tests/conftest.py

```diff
@@ -12,6 +13,18 @@
 from cachito.web.config import TEST_DB_FILE
 from cachito.web.app import create_app
 
+try:
+    from tests.temp_env_var import TEMP_ENV_VARS, ENV_VARS_TO_SUSPEND
+except ImportError:
+    TEMP_ENV_VARS = {}
+    ENV_VARS_TO_SUSPEND = []
+
+
+def pytest_configure(config):
```

Comment:
Instead of doing this, shall we rather provide a default value if env var is not specified?

Reply:
We are providing a default value in the temp_env_var.py file.  Also changing it to .tox/.tmp to avoid polluting the developer's file system.